obspy/imaging/spectrogram.py

```diff
@@ -102,7 +103,21 @@ def spectrogram(data, samp_rate, per_lap=0.9, wlen=None, log=False,
     :param clip: adjust colormap to clip at lower and/or upper end. The given
         percentages of the amplitude range (linear or logarithmic depending
         on option `dbscale`) are clipped.
+    :type starttime: :class:`~obspy.core.utcdatetime.UTCDateTime`
+    :param starttime: The starttime of the data array can be provided to use
+        absolute values on the time axis. This can be useful to combine
+        multiple spectrogram calls in the same axes.
     """
+    if axes:
+        if log and axes.images:
+            msg = ("Combinations of spectrogram plots with `log=True` "
+                   "and `log=False` ar not possible.")
```

Comment:
Huh.. is that change even related to the PR title.. ? Seems like an unrelated fix I put in here..